CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Coding style
+
+* like HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+# Branch policy
+
+## `master` branch
+
+`master` is the branch where every feature is available, but not necessarily finalized:
+ - Can cheat, but has to be avoided (allowed only in a small number of modules, which are explicitly marked unstable, no-cheat CI test mentioned below would need a corresponding exception list)
```

Comment:
I didn't find anything about **no-cheat** CI tests specifically. Did you forget it?

Reply:
no, but it wasn't as wordy as you expected it possibly :)